sources/iTermProfilePreferences.m

```diff
@@ -77,9 +88,13 @@ + (BOOL)defaultValueForKey:(NSString *)key isCompatibleWithType:(PreferenceInfoT
     id defaultValue = [self defaultValueMap][key];
     switch (type) {
         case kPreferenceInfoTypeIntegerTextField:
+        case kPreferenceInfoTypeUnsignedIntegerTextField:
```

Comment:
I think this should go with UnsignedIntegerPopup since a double cannot precisely represent every possible NSUInteger.